dist/api_data.py

```diff
@@ -750,6 +750,10 @@ def __ge__(self, other):
                 eviction to mark the number of obsolete time window pages that are marked as dirty
                 per btree in a single checkpoint''',
                 min=0, max=100000),
+            Config('cc_obsolete_tw_pages_dirty', '100', r'''
```

Comment:
Can we avoid using checkpoint cleanup in the api? We will rename the checkpoint cleanup to something else in the future?

Reply:
There are a lot of stats with checkpoint cleanup, I do not think that the name will go away soon, If we are doing, we need to change everything, at that time changing this also shouldn't be a problem IMO.

Reply:
Should we have added a prefix to the existing `obsolete_tw_pages_dirty` so we know it's eviction? i.e `evict_obsolete_tw_pages_dirty`?

Reply:
Fixed.